cloud/edgecontroller/hack/update-codegen.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
is this script an auto-generated file? If no can we rename to something related to device CRD as it looks like it is specific to that feature.

Reply:
This script is for code generation and not specific to crd.

Reply:
It is not auto generated

Reply:
ok, thanks